### DISCUSSION TITLE
Slow API calls

### DISCUSSION THREAD

**Discussion by prasanthbattula** [vercel/next.js#28911] [Help]
I have a API developed with Django Rest API and hosted in AWS EC2. I have my database in AWS RDS. Both my EC2 and RDS are in Mumbai (Asia Pacific - South 1) Region. Front developed with Next JS and deployed at Vercel. 
My API calls are taking long time.

What might be the problem?

**Comment by wwwdepot** [comment]
You can change deployment region to Mumbai (ap-south-1), but you need to be on pro plan to do that. See [docs](https://vercel.com/docs/serverless-functions/regions).

**Reply by prasanthbattula** [reply]
I have changed the region and redeployed the source code. Now I am getting "**Application error: a client-side exception has occurred (see the browser console for more information)**" The console show the issue with api calls. I am on **Pro** trail plan.
